src/libs/InputUtils/index.web.ts

```diff
@@ -0,0 +1,11 @@
+function scrollToBottom(input: HTMLInputElement) {
```

Comment:
We can create function type the same `updateMultilineInputRange/types.ts`  to avoid different parameter on each platforms

Reply:
If we do that, then we need to check for scrollTop again because the input could be HTMLInputelement | TextInput. They have different type because they are different.
```
if ('scrollTop' in input) ...
```

Reply:
Yes please do that, then we can remove disable type check for [InputUtils/index.ts](https://github.com/Expensify/App/pull/51516/files#diff-b9486070dc3f6b0a3f998b2f466db1c92662fe8e6652b3d3eae231bde8a76ef3)

Reply:
Sorry, not sure if I get it.

> // eslint-disable-next-line @typescript-eslint/no-unused-vars

I added this because the param in the index.ts file is unused. Let me know if I missed what you mean.

Reply:
After create function type we can do like that to avoid disable eslint
```typescript
// types.ts
export type ScrollToBottom = (input: HTMLInputElement | TextInput) => void;
// index.ts
const scrollToBottom: ScrollToBottom = () => {};
```

Reply:
Done. I renamed the file too because the function now accept `HTMLInputElement | TextInput`